lib/keyword_extractor.js

```diff
@@ -1,6 +1,6 @@
 var _ = require("underscore");
 _.str = require('underscore.string');
-var supported_languages = ["english","spanish","polish", "german", "french", "italian", "dutch", "russian"];
+var supported_languages = ["english","spanish","polish", "german", "french", "italian", "dutch", "russian", "portuguese", 'portugueseTwo'];
```

Comment:
It doesn't need two separate Portuguese entries. I was testing a new model of stopwords. @michaeldelorenzo